### PR TITLE
First commit to the fibonacci without recursive task

### PR DESCRIPTION
**Pull request by PszemekZ**

### CODE REVIEW

File: fibonacci-without-recursive.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Napisz funkcję fibonacci(n), która, nie używając rekurencji, zwróci n-ty wyraz Ciągu Fibonacciego.
+
+function fibonacci(n) {
+    if (n < 2) {
+        return n;
+    }
+
+    let a = 0,
+        b = 1,
+        c = 1;
```
Comment (dabrowskisoftware) [inline fibonacci-without-recursive.js]: It this line necessary?

**Review by dabrowskisoftware — CHANGES_REQUESTED** [review]
It's working but... a, b, c?

Please start using descriptive names for variables.

File: fibonacci-without-recursive.js (hunk 1 above)
Comment (PszemekZ) [reply fibonacci-without-recursive.js]: Yes, because the value of variable has to be always the same in this task. Of cours this is my opinion

Comment (dabrowskisoftware) [reply fibonacci-without-recursive.js]: Before you write something like this, you should remove the line and check if anything has changed. I can guarantee you that you don't need this line. Please, remove it and try to understand why you don't need it.

File: fibonacci-without-recursive.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+function fibonacci(n) {
+    if (n < 2) {
+        return n;
+    }
+
+    let currentNumber = 0,
+        previousNumber = 1;
+
+    for (let i = 2; i <= n; i++) {
+        nextNumber = currentNumber + previousNumber;
```
Comment (dabrowskisoftware) [inline fibonacci-without-recursive.js]: This variable is not declared.

**Review by dabrowskisoftware — CHANGES_REQUESTED** [review]

**Review by dabrowskisoftware — APPROVED** [review]

**Pull request merged**